hack/generate/gen-test-framework.sh

```diff
@@ -13,7 +13,7 @@ cd test/test-framework
 
 # Ensure test-framework is up-to-date with current Go project dependencies.
 echo "$(../../build/operator-sdk print-deps)" > go.mod
-sed -i".bak" -E -e "s|github.com/operator-framework/operator-sdk[[:blank:]]+master||g" go.mod; rm -f go.mod.bak
+sed -i".bak" -E -e "/github.com\/operator-framework\/operator-sdk .+/d" go.mod; rm -f go.mod.bak
```

Comment:
Why the `| -> /` change?

Reply:
It seems only sed's `s` command supports different pattern delimiters. I switched to no longer substitute, just delete the line.